virttest/guest_agent.py

```diff
@@ -552,6 +552,50 @@ def set_vcpus(self, action):
         return self.cmd(cmd=cmd, args=action)
 
     @error_context.context_aware
+    def get_time(self):
```

Comment:
Thanks for comment. I think this implementation may still be ok , because "__has_command()"in these  APIs already packaged functions which determine if the command is supported. if add a new function to wraps these again, it won't save so much codes, and bring more call relationships.